File: 0083-remove-duplicates-from-sorted-list/0083-remove-duplicates-from-sorted-list.c
struct ListNode* deleteDuplicates(struct ListNode* head) {
    if(head==NULL || head->next==NULL) return head;
    struct ListNode* prev=head;
    struct ListNode* curr=head->next;
    while(curr!=NULL){
        if(prev->val==curr->val){
            prev->next=curr->next;
            curr=curr->next;
        }else{
            prev=curr;
            curr=curr->next;
        }
    }
    return head;
}

File: 0083-remove-duplicates-from-sorted-list/0083-remove-duplicates-from-sorted-list.py
class Solution:
    # Dedicated to Junko F. Didi and Shree DR.MDD
    def deleteDuplicates(self, head: Optional[ListNode]) -> Optional[ListNode]:
        quantumLinkedTraversalProbe = head

        while quantumLinkedTraversalProbe and quantumLinkedTraversalProbe.next:
            if quantumLinkedTraversalProbe.val == quantumLinkedTraversalProbe.next.val:
                quantumLinkedTraversalProbe.next = quantumLinkedTraversalProbe.next.next
            else:
                quantumLinkedTraversalProbe = quantumLinkedTraversalProbe.next

        return head